action.yml

```diff
@@ -84,14 +84,59 @@ branding:
   color: yellow
   icon: upload-cloud
 runs:
-  using: docker
-  image: Dockerfile
-  args:
-  - ${{ inputs.user }}
-  - ${{ inputs.password }}
-  - ${{ inputs.repository-url }}
-  - ${{ inputs.packages-dir }}
-  - ${{ inputs.verify-metadata }}
-  - ${{ inputs.skip-existing }}
-  - ${{ inputs.verbose }}
-  - ${{ inputs.print-hash }}
+  using: composite
+  steps:
+  - name: Reset path if needed
+    run: |
+      # Reset path if needed
```

Comment:
Why would this be needed outside the container?

Reply:
Because you've set up a test that modifies the `$PATH` (https://github.com/pypa/gh-action-pypi-publish/issues/112, 1350b8bd724c983bdf030b52cba047a6eea5dc6d).

https://github.com/pypa/gh-action-pypi-publish/blob/699cd6103f50bf5c3b2f070c70712d109c168e6c/.github/workflows/self-smoke-test-action.yml#L85-L90

Reply:
Ack. Marking as resolved.